tests/unittests/Foundation/NSComparisonPredicateTests.m

```diff
@@ -630,7 +630,7 @@ - (void)dealloc {
 
     ASSERT_TRUE_MSG(comparisonPredicate != nil, "FAILED: comparisonPredicate should be non-null!");
 
-    ASSERT_TRUE_MSG([comparisonPredicate evaluateWithObject:nil substitutionVariables:nil], "FAILED: Comparison failed.");
+    ASSERT_TRUE_MSG(![comparisonPredicate evaluateWithObject:nil substitutionVariables:nil], "FAILED: Comparison failed.");
```

Comment:
are vs Are

---
In reply to: [73914570](https://github.com/Microsoft/WinObjC/pull/846#discussion_r73914570) [](ancestors = 73914570)